cli/internal/storage/files/files.go

```diff
@@ -60,16 +60,28 @@ func list(ctx context.Context) error {
 		return err
 	}
 
-	var req fv1.ListFilesRequest
-	var resp fv1.ListFilesResponse
-	if err := ihttp.NewClient(env).Send(http.MethodGet, path, &req, &resp); err != nil {
-		return err
+	var files []*fv1.File
+	var after string
+	for {
+		req := fv1.ListFilesRequest{
+			After: after,
+			Limit: 1,
```

Comment:
Would this mean listing the files one by one?

Reply:
Ah sorry, this was just for testing. Let's me remove.